ui/src/containers/AppVersionsTableContainer.js

```diff
@@ -9,20 +9,42 @@ import './TableContainer.css';
 import config from '../config/config';
 
 export class AppVersionsTableContainer extends React.Component {
+  state = {
```

Comment:
if using local state like this, do we need a constructor?

Reply:
There is no constructor here.

Reply:
Yes, but shouldn't we be initialising local state in our class component? If so, we [should need](https://reactjs.org/docs/react-component.html#constructor) a constructor to do so

Reply:
Oh I think I misunderstood. No I don't believe we need a constructor.

Reply:
https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599

Reply:
Cheers, thanks for the link - hadn't realised that, thanks for the link, seems we can do without them! Woohoo - less code :D